cloudera-integration/parcel/conf/solr-configs/pulseconfig-securev1/schema.xml

```diff
@@ -19,20 +19,20 @@
 <schema name="pulse" version="1.0">
 
     <fields>
-        <field name="id" type="string" indexed="true" stored="true" required="true"
+        <field name="id" type="string" indexed="true" stored="true" required="false"
```

Comment:
I _think_ we need to keep `id` required (but only `id`), because it's the unique key. The key is auto generated. I'm looking here: https://wiki.apache.org/solr/UniqueKey. But since it's auto generated, we can just make it required and not worry about it.